docs/sources/clients/promtail/scraping.md

```diff
@@ -209,7 +211,11 @@ Before using `gcplog` target, GCP should be [configured](../gcplog-cloud) with p
 
 It also support `relabeling` and `pipeline` stages just like other targets. 
 
-When Promtail receives GCP logs the labels that are set on the GCP resources are available as internal labels. Like in the example above, the `__project_id` label from a GCP resource was transformed into a label called `project` through `relabel_configs`. See [Relabeling](#relabeling) for more information.
+When Promtail receives GCP logs various internal labels are made available for [relabeling](#relabeling):
+  - `__gcp_logname`
+  - `__gcp_resource_type`
+  - `__gcp_resource_labels_<NAME>`
+    In the example above, the `project_id` label from a GCP resource was transformed into a label called `project` through `relabel_configs`.
```

Comment:
```suggestion
In the example above, the `project_id` label from a GCP resource was transformed into a label called `project` through `relabel_configs`.
```

Reply:
This prose intentionally belongs to the `__gcp_resource_labels_<NAME>` list item.

Reply:
Yes, unfortunately without a preview I can't confirm that the indentation will work as expected here. Often, these types of indented content lose that format, thus my suggestion to simply remove the tab. It's by no means a critical fix.

Reply:
In any compliant markdown it should work.
If it doesn't, then worst case it falls back to your suggested change.